src/transport/websocket.ts

```diff
@@ -528,7 +527,7 @@ export default class WebSocketTransport implements SubscriptionTransport {
     const [receviedPongID] = message;
 
     if (this.lastSentPingID !== receviedPongID) {
```

Comment:
We removed this being a connection termination protocol error previously in a panic. Should be be reinstating that?